cmake/options.cmake

```diff
@@ -3,9 +3,9 @@ set (PROJECT_VER_MINOR  1)
 set (PROJECT_VER_PATCH  0)
 set (PROJECT_VER        0.1.0)
 
-if (NOT CMAKE_BUILD_TYPE)
+if (NOT DEFINED CMAKE_BUILD_TYPE)
   set (CMAKE_BUILD_TYPE Debug)
-endif (NOT CMAKE_BUILD_TYPE)
+endif (NOT DEFINED CMAKE_BUILD_TYPE)
```

Comment:
I think  both condition above CMAKE_INSTALL_LIBDIR && CMAKE_INSTALL_BINDIR should be reworked also. 
anyway let's fix this first